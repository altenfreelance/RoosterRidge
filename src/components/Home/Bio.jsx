import React from 'react';

export default function Bio() {
    return (
        <div>
            <h1>About the Band</h1>
            <div className="card py-4 px-5">
                <p>
                    Blending cutting-edge bluegrass and folk with the dulcet tones of traditional Americana, Rooster Ridge is an ascendant voice in the Ohio
                    music scene.  Reminiscent of classic groups like The Band, The Stanley Brothers, Nitty Gritty Dirt Band, and more recently Old Crow
                    Medicine Show, Rooster Ridge brings together superior instrumentation, contagious melodies, harmonies as smooth as Kentucky bourbon,
                    and lyrics as frank and honest as the countryside to offer a distinctive new sound that still holds firm to its roots.
                </p>
                <p>
                    Rooster Ridge was founded in Loveland, Ohio by the Alten Brothers, Jake and Andrew, and the father-son team of Greg and Vince Stevens.
                    Rounding out the group are close friends and lead vocalists Joey Oberholzer and Bryce Clawson, all anchored by Dave Blumbergâ€™s
                    foundational bass lines.
                </p>
                <p>
                    Starting out quite literally as a late-night, back-porch jam session, Rooster Ridge has begun to catch fire, drawing ever larger
                    crowds and an exploding local fan base as they play a growing universe of venues.  Rooster Ridge thanks all of their supporters
                    for an exceptional start to what many listeners believe will be a promising and prolific future.  Rooster Ridge invites you to sit
                    back, relax, and become part of their distinctly American musical journey.
                </p>

            </div>

        </div>
    )
}