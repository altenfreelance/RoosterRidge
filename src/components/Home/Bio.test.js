import React from 'react';
import Bio from './Bio';

import {render, cleanup} from '@testing-library/react';
import '@testing-library/jest-dom';

afterEach(cleanup);

it('renders (matches snapshot) ', () => {
    const {asFragment} = render(<Bio />);
    expect(asFragment()).toMatchSnapshot();
});
