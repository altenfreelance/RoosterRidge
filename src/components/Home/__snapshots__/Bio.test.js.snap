// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders (matches snapshot)  1`] = `
<DocumentFragment>
  <div>
    <h1>
      About the Band
    </h1>
    <div
      class="py-4 px-5"
      style="color: white;"
    >
      <p>
        Blending cutting-edge bluegrass and folk with the dulcet tones of traditional Americana, Rooster Ridge is an ascendant voice in the Ohio music scene.  Reminiscent of classic groups like The Band, The Stanley Brothers, Nitty Gritty Dirt Band, and more recently Old Crow Medicine Show, Rooster Ridge brings together superior instrumentation, contagious melodies, harmonies as smooth as Kentucky bourbon, and lyrics as frank and honest as the countryside to offer a distinctive new sound that still holds firm to its roots.
      </p>
      <p>
        Rooster Ridge was founded in Loveland, Ohio by the Alten Brothers, Jake and Andrew, and the father-son team of Greg and Vince Stevens. Rounding out the group are close friends and lead vocalists Joey Oberholzer and Bryce Clawson, all anchored by Jamie Davidsonâ€™s foundational bass lines.
      </p>
      <p>
        Starting out quite literally as a late-night, back-porch jam session, Rooster Ridge has begun to catch fire, drawing ever larger crowds and an exploding local fan base as they play a growing universe of venues.  Rooster Ridge thanks all of their supporters for an exceptional start to what many listeners believe will be a promising and prolific future.  Rooster Ridge invites you to sit back, relax, and become part of their distinctly American musical journey.
      </p>
    </div>
  </div>
</DocumentFragment>
`;
